source/languageHandler.py

```diff
@@ -95,8 +96,13 @@ def getAvailableLanguages(presentational=False):
 	@rtype: list of tuples
 	"""
 	#Make a list of all the locales found in NVDA's locale dir
-	locales = [x for x in os.listdir('locale') if not x.startswith('.')]
-	locales = [x for x in locales if os.path.isfile('locale/%s/LC_MESSAGES/nvda.mo'%x)]
+	localesDir = os.path.join(globalVars.appDir, 'locale')
+	locales = [
+		x for x in os.listdir(localesDir) if (
+			not x.startswith('.')

```

Comment:
Any idea why this is here? So languages can be disabled? Or because there may have been "dot files" in this directory?

If you know a comment would be good.